lib/ui/urlbar.js

```diff
@@ -33,9 +33,10 @@ Urlbar.prototype = {
     delete this.el;
     delete this.win;
   },
+  // Set the URL in the urlbar to either data.newUrl, if it's defined;
+  // otherwise, reset the URL to the actual value the user typed.
```

Comment:
Grammar police :oncoming_police_car:: "either" should always be used with "or".

> Set the URL in the urlbar data.newUrl, if defined; otherwise reset the URL to the actual value the user typed.